sunpy/tests/figure_hashes_mpl_dev_ft_261_astropy_dev.json

```
@@ -32,6 +32,7 @@
   "sunpy.map.tests.test_plotting.test_heliographic_quadrangle_width_height": "f1dc10238c83fbdf510d1f912c921fe3b53f437f9a46f3b71ad46b48273369fb",
   "sunpy.map.tests.test_plotting.test_heliographic_quadrangle_top_right": "b4d2229be76ed1fdcfa5740aaeb26ab6eed92608661cf2f34bca3685f524caf1",
   "sunpy.map.tests.test_plotting.test_heliographic_grid_annotations": "eae3267f21837571a7b10bf1e2fe0f672cf59c11ebefa4e5eadc3e9906609be3",
+  "sunpy.net.tests.test_helioviewer.test_download_jp2_map": "438fdcccc6f879491c876ce24a591843bb60d724e354d1c8124678a356ff3426",
```

```suggestion
  "sunpy.net.tests.test_helioviewer.test_download_jp2_map": "67b17ac7c343a1d568d9dfa2bb12e0041e2fc53e8ca1c0e1c2c7693eda2bd0d8",
```
This is what I got locally.

---

My apologies, might've mixed it up.